rclcpp/include/rclcpp/client.hpp

```diff
@@ -39,6 +39,9 @@
 #include "rmw/error_handling.h"
 #include "rmw/rmw.h"
 
+#include "rcutils/logging_macros.h"
```

Comment:
Nit: we encourage the use of single vertical space

Reply:
fixed